turbopack/crates/turbo-persistence/src/write_batch.rs

```diff
@@ -420,8 +420,19 @@ impl<K: StoreKey + Send + Sync, const FAMILIES: usize> WriteBatch<K, FAMILIES> {
         let seq = self.current_sequence_number.fetch_add(1, Ordering::SeqCst) + 1;
         let mut buffer = Vec::new();
         buffer.write_u32::<BE>(value.len() as u32)?;
-        lz4::compress_to_vec(value, &mut buffer, ACC_LEVEL_DEFAULT)
-            .context("Compression of value for blob file failed")?;
+        let compressed_bound =
```

Comment:
Last I recall in discussions with @sokra was that we set the blob threshold so high that in practice nothing really uses blobs and everything is in sst files.